EIPS/eip-4788.md

```diff
@@ -158,24 +162,29 @@ push0
 revert
 
 jumpdest
-push3 0x018000
 push0
 calldataload
-mod
 dup1
+iszero
+push1 0x49
+jumpi
+
+push3 0x016da0
+dup2
+mod
+swap1
+dup2
 sload
-push0
-calldataload
 eq
-push1 0x37
+push1 0x3c
 jumpi
 
 push0
 push0
 revert
 
 jumpdest
-push3 0x018000
+push3 0x016da0
```

Comment:
This is the part of the bytecode that was mentioned on ACD, it should be the same number as mentioned in line 32.